inject-java/src/main/java/io/micronaut/annotation/processing/BeanDefinitionInjectProcessor.java

```diff
@@ -277,6 +282,20 @@ public final boolean process(Set<? extends TypeElement> annotations, RoundEnviro
         return false;
     }
 
+    private static Element resolvedFailedElement(Object errorElement) {
```

Comment:
would it be possible to do this wherever the PostponeToNextRoundException is created? the other use of getErrorElement seems to be doing something similar